documents/Libraries/stdlib/impl/shadergen/osl/impl.mtlx

```diff
@@ -36,35 +36,6 @@
   </implementation>
 
```

Comment:
Many of our node implementations were referencing source code from SPI's reference OSL implementation. That code doesn't work out of the box in ShaderX. I have added in source code for many of these here, and for the remaining ones I removed the their impl elements, since our test for missing implementations will otherwise not report them as missing.